src/openapi_server/apis/reading_api.py

```diff
@@ -18,25 +19,33 @@
 from openapi_server.models.extra_models import TokenModel  # noqa: F401
 from openapi_server.security_api import get_token_basic
 
+from openapi_server.connect import client_for_basic_auth
+
 router = APIRouter()
 
 
 @router.get(
     "/judgment/{judgmentUri:path}",
     responses={
         200: {"description": "A single judgment document, in Akoma Ntoso XML"},
+        404: {"description": "Document not found"},
     },
     tags=["Reading"],
     summary="Read a judgment or decision, given its URI",
     response_model_by_alias=True,
 )
 async def get_document_by_uri(
+    response: Response,
     judgmentUri: str = Path(None, description=""),
     token_basic: TokenModel = Security(get_token_basic),
-) -> str:
-    """Unless the client has `read_unpublished_documents` permission,
-    then only published documents are accessible."""
-    return "jqqzdwd"
+):
+    try:
+        client = client_for_basic_auth(token_basic)
+        judgment = client.get_judgment_xml(judgmentUri)
+    except MarklogicResourceNotFoundError:
```

Comment:
We should be able to replace this with a general `except MarklogicError` now, but this is fine.